### DISCUSSION TITLE
Keep getting 404 Not found for some of routes only happen to v2.10.8

### DISCUSSION THREAD

**Discussion by arsatali91** [livewire/livewire#5470] [7. Bugs (archived)]
<img width="1419" alt="image" src="https://user-images.githubusercontent.com/61260172/209767558-eb87bed7-0dd1-4fac-b901-41eaa0ff9660.png">
After I switched to v2.10.7 , it seems fixed. help @calebporzio ✋ .

**Comment by lam0819** [comment]
Could you check, is it related to  https://github.com/livewire/livewire/pull/4570

**Reply by Simian-m** [reply]
Yes it is. The fix @Shtier proposed on that issue is also working for people with a subdirectory URL structure.

**Comment by Guillermocen** [comment]
since last week i have been stuck with the same problem with livewire version 2.10.8, i tried it on a fresh laravel 8 and laravel 9 project, resulting in the same problem, i am getting 404 error when executing a simple method with `wire:click="exampleMethod" ` and in the component class, said method:
```
public function exampleMethod() {
  dd('ok');
}
```
It is a full page component, in the web.php routes file I only have:
`Route::get('/', App\Http\Livewire\ExampleComponent::class)`

I had to change to version 2.10.7 so that everything could work correctly, but, I would like to know in detail, the reason for this problem in the latest version (2.10.8) of livewire, for now I could not solve it for myself myself, so I would like your support to understand and fix it. Thank you

**Reply by lam0819** [reply]
this should be related to https://github.com/livewire/livewire/pull/4570

**Reply by Guillermocen** [reply]
Does it mean that from now on and for the next versions, do you suggest applying this request to the original code of the package or is it momentary while they fix it for the next version?

**Comment by joshhanley** [comment]
@arsatali91 and @Guillermocen there was a bug released in v2.10.8. I have submitted a PR #5490 with a fix. Hope this helps!
